#constants for Standby State Actions
uint8 WAIT = 0
uint8 START_NEW_MISSION = 1
uint8 LOAD_FILES = 2

#request
uint8 stby_action
---
bool result